enterprise/utils.py

```diff
@@ -166,3 +171,97 @@ def get_catalog_admin_url_template():
 
     # Return matched FQDN from catalog api url appended with catalog admin path
     return match.group("fqdn").rstrip("/") + "/admin/catalogs/catalog/{catalog_id}/change/"
+
+
+def send_notification_message(
+        request,
+        user,
+        course,
+        course_url,
+        course_type,
+        organization,
+        start_date,
+        program_type=None,
+        email_connection=None,
+):
+    """
+    Send an email notifying a user about their enrollment in a course.
+
+    Arguments:
+        user: Either a User object or a PendingEnterpriseCustomerUser that we can use
+            to get details for the email
+        course: The name of the course or program that the user's been enrolled in
+        course_url: The URL to the LMS details page for the course or program.
+        course_type: The classification of the learning unit ("course" or "program")
+        organization: The name of the organization sponsoring the enrollment
+        start_date: The date the course first becomes available
+        program_type: An additional classifier for the type of the course - for example,
+            a program could be a "MicroMaster" program
+        email_connection: An existing Django email connection that can be used without
+            creating a new connection for each individual message
+    """
+    if hasattr(user, 'first_name') and hasattr(user, 'username'):
+        # PendingEnterpriseCustomerUsers don't have usernames or real names. We should
+        # template slightly differently to make sure weird stuff doesn't happen.
+        user_name = user.first_name
+        if not user_name:
+            user_name = user.username
+    else:
+        user_name = None
+
+    # Users have an `email` attribute; PendingEnterpriseCustomerUsers have `user_email`.
+    if hasattr(user, 'email'):
+        user_email = user.email
+    elif hasattr(user, 'user_email'):
+        user_email = user.user_email
+    else:
+        raise TypeError(_('`user` must have one of either `email` or `user_email`.'))
+
+    msg_context = {
+        'user_name': user_name,
+        'course_name': course,
+        'program_type': program_type,
+        'course_type': course_type,
+        'organization_name': organization,
+        'start_date': start_date,
+        'course_url': course_url
+    }
+    try:
+        current_site = get_current_site(request)
```

Comment:
Right now, it'd send from edx.org - good catch. Possibly grab the correct site off the EnterpriseCustomer?